src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -0,0 +1,2103 @@
+
+Both enrollment and event can have as many notes as needed - there is no limit. However, it is not possible to delete or update neither of these notes. They are like a logbook. If one wants to amend a note, can do so by creating another note. The only way to delete a note is by deleting the parent object - either event or enrollment. 
+
+Tracker notes do not have their own dedicated endpoint, they are exchanged as part of the parent event and/or enrollment payload. Below is a sample payload.
+
+```json
+{
+  "trackedEntityInstance": "oi3PMIGYJH8",
+  <entity_details>,
+  ],
+  "enrollments": [
+    {
+      "enrollment": "EbRsJr8LSSO",
+      <enrollment_details>
+      "notes": [
+        {
+          "note": "vxmCvYcPdaW",
+          "value": "Enrollment note 2.",
+        },
+        {
+          "value": "Enrollment note 1",
+        }
+      ],
+      
+      "events": [
+        {
+          "event": "zfzS9WeO0uM",
+          <event_details>,
+          "notes": [
+            {
+              "note": "MAQFb7fAggS",
+              "value": "Event Note 1.",
+            },
+            {
+              "value": "Event Note 2.",
+            }
+          ],
+        },
+        {
+          ...
+        }
+      ]
+    }
+  ]
+}
+```
+
+
+| Property | Description | Required | Immutable | Type | Example |
+|---|---|---|---|---|---|
+| note | The reference of the note. Generated if empty | No | Yes | String:Uid | ABCDEF12345 |
+| value | The content of the note. | Yes | Yes | String:Any | This is a note |
+| storedAt | Timestamp when the user added the note. Set on the server. | No | Yes | Date:ISO 8601 | YYYY-MM-DDThh:mm:ss |
+| updatedAt | Timestamp when the note was last updated. Set on the server. | No | Yes | Date:ISO 8601 | YYYY-MM-DDThh:mm:ss |
+| storedBy | Client reference for who stored/created the note. | No | No | String:Any | John Doe |
+
+## Tracker Import (`POST /api/tracker`)
+<!--DHIS2-SECTION-ID:webapi_nti_import-->
+
+The `POST /api/tracker` endpoint allows clients to import the following tracker objects into DHIS2:
+
+* **Tracked entities**
+* **Enrollments**
+* **Events**
+* **Relationships**
+* Data embedded in other [tracker objects](#webapi_nti_tracker_objects)
+
+Main changes compared to version 2.35 are:
+1. Import payload can be ***nested*** or ***flat***
+2. Invocation can be ***synchronous*** or ***asynchronous***
+
+### Request parameters
+
+Currently tracker import endpoint support the following parameters:
+
+#### ***SYNC*** endpoint
```

Comment:
The section seems to list all available request params for both sync and async - maybe we should rename it?

Reply:
I rewrote this :)